package.json

```diff
@@ -44,7 +44,7 @@
     "rxjs": "5.4.2",
     "style-loader": "0.18.2",
     "sw-precache-webpack-plugin": "0.11.4",
-    "transifex": "1.4.6",
+    "transifex": "git://github.com/mattkime/node-transifex.git#d109c7c",
```

Comment:
I recommend sourcing Meetup-related PRs from `github.com/meetup/` repos in the future, just so all the update permissions can be managed internally in case you got sick or whatever.